idd/Energy+.idd.in

```diff
@@ -17807,6 +17819,175 @@ SurfaceProperty:SolarIncidentInside,
        \required-field
        \type object-list
        \object-list ScheduleNames
+	   
+SurfaceProperty:LocalEnvironment,
```

Comment:
Why are there no memos on this object or the next?  It is critical that these new objects are clearly defined so users don't attempt to use them incorrectly.  There should be \memo fields, \min-fields, and in the case of this particular object, should any of A2-A5 be required??